src/Spago/Command/Ls.purs

```diff
@@ -45,6 +53,18 @@ type LsEnv =
   , selected :: WorkspacePackage
   }
 
+listPaths :: Spago { logOptions :: LogOptions } Unit
+listPaths = do
+  logDebug "Running `listPaths`"
+  traverse_ logInfo
+    [ "Global cache path: " <> Paths.globalCachePath
+    , "Global registry path: " <> Paths.registryPath
+    , "Global registry index path: " <> Paths.registryIndexPath
+    , "Global package sets path: " <> Paths.packageSetsPath
+    , "Local cache path: " <> Paths.localCachePath
+    , "Local cache packages path: " <> Paths.localCachePackagesPath
+    ]
```

Comment:
Could we add the db path here as well? We'll need to move it from `Spago.Db.databasePath` to here